index.html

```diff
@@ -83,23 +83,23 @@ <h6>You can control how the viewer looks for navPlace properties.</h6>
         </div>
         <footer class="nav nav-center text-primary is-fixed is-full-width, is-vertical-align">
             <div>
-                <a target="_blank" href="https://www.slu.edu/research/faculty-resources/research-computing.php">
-                    <img class="brand" src="./images/rcg-logo.jpg"/> <small>&copy;2023 Research Computing Group </small>
+                <a target="_blank" class="tooltip" href="https://www.slu.edu/research/faculty-resources/research-computing.php">
+                    <img class="brand" src="./images/rcg-logo.jpg"/> <small>&copy;2023 Research Computing Group </small><span class="tooltiptext">&copy;2023 Research Computing Group</span>
                 </a>
             </div>
             <div>
-                <a target="_blank" href="https://rerum.io">
-                    <img class="brand" src="./images/rerum-logo.png"/> <small>RERUM</small>
+                <a target="_blank" class="tooltip" href="https://rerum.io">
+                    <img class="brand" src="./images/rerum-logo.png"/> <small>RERUM</small><span class="tooltiptext">RERUM</span>
```

Comment:
These `tooltiptext` elements never appear.  If you are trying to cause a tooltip effect, it did not work.  There is no tool tip when I hover over these (using Google Chrome browser on computer). 

See https://www.w3schools.com/tags/tag_title.asp